docker/update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+docker build ./docker -f docker/Dockerfile.debian-buster-aarch64 \
+    -t ghcr.io/qwandor/cross-dbus-debian-buster-aarch64:latest
+docker build ./docker -f docker/Dockerfile.debian-buster-armv7 \
+    -t ghcr.io/qwandor/cross-dbus-debian-buster-armv7:latest
+docker build ./docker -f docker/Dockerfile.debian-buster-x86_64 \
+    -t ghcr.io/qwandor/cross-dbus-debian-buster-x86_64:latest
+
+docker push ghcr.io/qwandor/cross-dbus-debian-buster-aarch64:latest
```

Comment:
maybe also tag this with a version number (defined as a constant at the top of the file?) and push that too? Then you can use this versioned image in Cross.toml and have a better chance at avoiding breaking the build later

Reply:
Done.